coconut.sh

```diff
@@ -0,0 +1,45 @@
+package: coconut
+version: "%(commit_hash)s"
+tag:  master
+build_requires:
+  - golang
+  - protobuf
+source: https://github.com/AliceO2Group/Control
+incremental_recipe: |
```

Comment:
Incremental does the same as the base recipe, please drop the whole incremental recipe.